notebooks/harmony subsetting/Harmony L2 Subsetter.ipynb

```diff
@@ -46,7 +46,7 @@
   },
```

Comment:
<p>We get the bounding box of the granule itself and then use that to subset; I think we actually want to use a bounding box that is slightly smaller (5% maybe?) instead of the actual bounding box of the granule</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/podaac/tutorials/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>why not just use the same lat/lon we previously used (bounding_box=-90,-45.75,90,-45) in the CMR search. This way we know what bounds we're always looking for? Think of this from a user perspective (<strong>this is a tutorial after all!</strong>) they want to specify their bounding box up front. </p>

Reply:
<p>doesn't work for all collections, i seen some collection yield 0 granules then we don't have any granule to run the rest of the notebook</p>